src/c++/devices/fua/dmFua.c

```diff
@@ -146,8 +146,8 @@ static struct target_type fuaTargetType = {
 /**********************************************************************/
 int __init fuaInit(void)
 {
-  STATIC_ASSERT(offsetof(FuaDevice, dev) 
-                == offsetof(CommonDevice, dev)); 
+  BUILD_BUG_ON(offsetof(FuaDevice, dev)
+               != offsetof(CommonDevice, dev));
```

Comment:
If you're going to unwrap the line, I  think you should unwrap the line in all five test devices.